src/Kunstmaan/NodeBundle/Controller/WidgetsController.php

```diff
@@ -176,16 +183,16 @@ protected function nodesToArray($locale, $rootNodes, $depth = 2)
         /** @var DomainConfiguration $domainconfig */
         $domainconfig = $this->get('kunstmaan_admin.domain_configuration');
         $isMultiDomain = $domainconfig->isMultiDomainHost();
-        $switchedHost = $domainconfig->getHostSwitched()['host'];
-        $switched = $domainconfig->getHost() == $switchedHost;
+        $switchedHost = $domainconfig->getHostSwitched();
+        $switched = $domainconfig->getHost() == $switchedHost['host'];
 
         $results = [];
 
         /** @var Node $rootNode */
         foreach ($rootNodes as $rootNode) {
             if ($nodeTranslation = $rootNode->getNodeTranslation($locale, true)) {
                 if ($isMultiDomain && !$switched) {
-                    $slug = sprintf("[%s:%s]", $switchedHost, "NT" . $nodeTranslation->getId());
+                    $slug = sprintf("[%s:%s]", $switchedHost['id'], "NT" . $nodeTranslation->getId());
```

Comment:
Wouldn't this be BC breaking?

Reply:
I don't think so, the current outputted format is "broken". It produces this output for example `[www.example.com:NT55]`. But this won't actually work for [the regex](https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/33519719dc36c6adc28caf5a56b294f4cae0750a/src/Kunstmaan/NodeBundle/Helper/URLHelper.php#L76) behind the `replace_url` twig function. It expects a format of [site_name:NT55].